domains/nino-os.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Robotic-Germany",
+        "email": "rocker.lukas.wrld.16@gmail.com"
+    },
+    "record": {
+        "URL": "nino-os.dev"
```

Comment:
Unable to access site.

Reply:
Can you please delete this website/file, I changed my mind and now don't want it anymore :)